docs/deprecated.md

```diff
@@ -14,6 +14,7 @@ A list of deprecated features and the version in which they were deprecated foll
 
 -   Support to NGSIv1.
 -   Support to Node.js v4 in iotagent-json 1.9.0. The use of Node.js v8 is highly recommended.
+-   Support to Node.js v10 in iotagent-json 1.17.0. The use of Node.js v12 is highly recommended.
```

Comment:
For the sake of completeness, maybe information about versions between v4 and v10 should be included (it should be easy to know looking at https://github.com/telefonicaid/iotagent-json/releases )... although maybe it doesn't worth the time.

Not sure how it's better...

Reply:
Fixed b1dccdc